src/OpenTelemetry.Instrumentation.Process/README.md

```diff
@@ -92,6 +92,20 @@ Gets the user processor time for this process.
 * [Process.PrivilegedProcessorTime](https://learn.microsoft.com/dotnet/api/system.diagnostics.process.privilegedprocessortime):
 Gets the privileged processor time for this process.
 
+### process.processors
+
+Process threads count.
```

Comment:
`Process threads count.` -> `Process processor count.`

Might be nice to also do like...

`Process processor (CPU) count.`

`Gets the number of processors (CPUs) available to the current process.`

Reply:
updated